src/test/java/org/apache/drillui/test/framework/testng/secure/login/LoginTest.java

```diff
@@ -16,23 +16,70 @@
  */
 package org.apache.drillui.test.framework.testng.secure.login;
 
-import org.apache.drillui.test.framework.initial.TestProperties;
+import org.apache.drillui.test.framework.initial.PropertiesConst;
+import org.apache.drillui.test.framework.steps.webui.AuthSteps;
+import org.apache.drillui.test.framework.steps.webui.ProfilesSteps;
+import org.apache.drillui.test.framework.steps.webui.QueryResultsSteps;
 import org.apache.drillui.test.framework.testng.secure.BaseSecureTest;
 import org.testng.annotations.Test;
-import org.apache.drillui.test.framework.steps.webui.AuthSteps;
-
-import static org.testng.Assert.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertFalse;
 
 public class LoginTest extends BaseSecureTest {
+  private AuthSteps AuthSteps = new AuthSteps();
+  private QueryResultsSteps QueryResultsSteps =  new QueryResultsSteps();
+  private ProfilesSteps ProfilesSteps = new ProfilesSteps();
+
   @Test(groups = {"functional"})
   public void testLogin() {
-    assertEquals(
-        AuthSteps.login(TestProperties.get("ADMIN_USER1_NAME"), TestProperties.get("ADMIN_USER1_PASSWORD")).getLogoutText(),
-        "Log Out (" + TestProperties.get("ADMIN_USER1_NAME") + ")", "Login failed");
+    assertEquals(AuthSteps.login(PropertiesConst.ADMIN_1_NAME, PropertiesConst.ADMIN_1_PASSWORD).getLogoutText(), "Log Out (" + PropertiesConst.ADMIN_1_NAME + ")", "Login failed");
   }
 
   @Test(groups = {"functional"})
   public void testLogout() {
     assertEquals(AuthSteps.logOut().getLoginText(), "Log In", "Logout failed");
   }
+
+  @Test(groups = {"functional"})
+  public void AdminsViewVsUsersViewInSecurityMode() throws InterruptedException {
+    //Step 1. Run query for Admin1
+    assertEquals(AuthSteps.login(PropertiesConst.ADMIN_1_NAME, PropertiesConst.ADMIN_1_PASSWORD).navigateQuery().submitQuery("SELECT * FROM cp.`employee" + ".json` LIMIT 20").getQueryStatus(),"COMPLETED");
```

Comment:
It'd be more readable if you split a methods chain to several lines:
```suggestion
    assertEquals(AuthSteps.login(PropertiesConst.ADMIN_1_NAME,
     PropertiesConst.ADMIN_1_PASSWORD).navigateQuery()
     .submitQuery("SELECT * FROM cp.`employee" + ".json` LIMIT 20")
     .getQueryStatus(), "COMPLETED");
```

Reply:
done